tests/rt.conf

```diff
@@ -166,3 +166,7 @@ COMPILE | CCPP=Y SUITES=FV3_GFS_2017_gfdlmp,FV3_GFS_2017_gfdlmp_noahmp
 COMPILE | CCPP=Y SUITES=FV3_GFS_2017_gfdlmp,FV3_GFS_2017_gfdlmp_noahmp                                                                   | standard    | jet.intel      | fv3         |
 
 RUN     | fv3_ccpp_gocart_clm                                                                                                            | standard    |                | fv3         |
+######################################################################################################################################################################################
```

Comment:
Is it possible to combine this COMPILE line with a previous one (for example where the original GFSv16beta suite is compiled) to avoid an extra compile step?

Reply:
Also, please add this regression test to `rt_gnu.conf` as well. @junwang-noaa @DusanJovic-NOAA

Reply:
Yes, I did it